dipimage/localshift.m

```diff
@@ -0,0 +1,141 @@
+%LOCALSHIFT   Shifts/warps an image by a shift vector per pixel/grid point
+%
+% SYNOPSIS:
+%  out = localshift(in, sv, osize, interp, coord)
+%
+% PARAMETERS:
+%  in:     2D image
+%  sv:     shift vector per pixel as a dip_image_array
+% %  osize:  output size
+% %          'Recut':    new size = bounding bound around shifted image
+% %          'FullSize': new size = old size + max(abs(shift vector))
+% %          'ConstFoV': new size = old size (with constant Field of View)
+%  interp: Interplotation option
+% %          'zoh': zero order hold, shift via 2D histogram and normalized convolution
+%          'bilinear': osize is ignored=ConstFoV, loop is done over output image
+%          '3-cubic': osize is ignored=ConstFoV, loop is done over output image
+%  coord:  optional coordinate grid image (2D) as meshgrid, if the shift vector is not
+%          per pixel as the output from findlocalshift
+%
+% NOTE:
+%  only implemented for 2D but could be nD
+%  'zoh' is currently not supported, and hence related 'osize' is gone.

```

Comment:
The other change I made was to remove this line and the references to `osize` in the documentation above. I think it might be more confusing than anything having some lines of documentation that describe something not documented. If we ever need to implement the 'zoh' method, we can go back and add these lines.